docs/guide/media.md

```diff
@@ -0,0 +1,167 @@
+# Working with images and media files
+
+There are a few ways to configure a field schema to allow editors to select images or other media files.
+
+1. Attachment fields
+2. Relationship fields to the image or file piece types
+3. Area fields with the image widget
+
+In addition to covering the main ways to allow editors to choose files for their content, we'll also look at how to use the data from each approach in templates. We'll focus on how developers would get the file URL plus critical image information for generating markup.
+
+## The attachment field option
+
+The [attachment field](/reference/field-types/attachment.md) is the direct route for uploading any type of file (e.g., image, PDF) to the file system. It is a fairly simple field that does not do much more than upload the file.
+
+Importantly, a file uploaded through an attachment field *will not appear in either the media library* or the general file manager. If the image or file is meant to be reused, this will not be best. However if the file should only be associated with one particular page or piece it can work well. Uploading resumés associated with job applicants is one such example.
+
+```javascript
+// modules/article/index.js
+module.exports = {
+  // ...
+  fields: {
+    add: {
+      fileUpload: {
+        label: 'File upload',
+        type: 'attachment',
+        fileGroup: 'office'
```

Comment:
If I'm reading https://a3.docs.apos.dev/reference/field-types/attachment.html#settings correctly, then this should be `group` not `fileGroup`.

(As an aside, naming this property `group` feels unclear to me. Are there records of the discussion around this naming decision?)

Reply:
In this case the reference page is wrong. In A2 it was `group`. It's `fileGroup` in A3.